torch/distributed/pipelining/__init__.py

```diff
@@ -14,7 +14,7 @@
     ScheduleInterleaved1F1B,
     ScheduleLoopedBFS,
 )
-from .PipelineStage import ManualPipelineStage, PipelineStage
+from .PipelineStage import _PipelineStageBase, ManualPipelineStage, PipelineStage
```

Comment:
accidental merge?

Reply:
oh i see it was intentional.  But is it allowed?  (RE rules about _ modules and packages)  @kwen2501 

maybe you should just import this from PipelineStage directly in the test file.

Reply:
removed it and importing it from PipelineStage directly